observability/logging/logging-6.2/log6x-loki-6.2.adoc

```diff
@@ -0,0 +1,48 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[leveloffset=+1]
+[id="log6x-loki-6-2"]
+= Storing logs with LokiStack
+:context: log6x-loki-6.2
+
+toc::[]
+
+You can configure a `LokiStack` CR to store application, audit, and infrastructure-related logs.
+
+include::snippets/log6x-loki-statement-snip.adoc[leveloffset=+1]
+
+include::modules/log6x-loki-sizing.adoc[leveloffset=+1]
+
+[id="prerequisites-6-2_{context}"]
+== Prerequisites
+
+* You have installed the {loki-op} by using the command-line interface (CLI) or web console.
+* You have a `serviceAccount` in the same namespace in which you created the `ClusterLogForwarder`.
+* The `serviceAccount` is assigned the `collect-audit-logs`, `collect-application-logs`, and `collect-infrastructure-logs` cluster roles.
+
+[id="setup-6-2_{context}"]
+== Core Set up and configuration
+
+Use role-based access controls, basic monitoring, and pod placement to deploy Loki.
+
+include::modules/log6x-loki-rbac-rules-perms.adoc[leveloffset=+1]
+include::modules/log6x-enabling-loki-alerts.adoc[leveloffset=+1]
+include::modules/log6x-loki-memberlist-ip.adoc[leveloffset=+1]
+include::modules/log6x-loki-retention.adoc[leveloffset=+1]
+include::modules/log6x-loki-pod-placement.adoc[leveloffset=+1]
+
+[id="performance-6-2_{context}"]
+== Enhanced reliability and performance
+
+Use the following configurations to ensure Loki’s reliability and efficiency in production.
+
+include::modules/log6x-identity-federation.adoc[leveloffset=+1]
+include::modules/log6x-loki-reliability-hardening.adoc[leveloffset=+1]
+include::modules/log6x-loki-restart-hardening.adoc[leveloffset=+1]
+
+[id="advanced-6-2_{context}"]
+== Advanced Deployment and Scalability
+*Specialized configurations for high availability, scalability, and error handling.*
```

Comment:
```suggestion
== Advanced deployment and scalability

To configure high availability, scalability, and error handling, use the following information.
```

- Capitalization is not needed for the whole heading.
- Is there a reason why line no. 44 is in bold? It's not providing a clear direction for users.

Reply:
I don't know why it was bold (i just ported and not wrote!). But you are right, it doesn't look good. Thanks for the suggested rewrite, it reads much better.